gemac/management.py

```diff
@@ -0,0 +1,165 @@
+from myhdl import block, Signal, intbv, always_seq, concat, now
+
+
+rx0, rx1, tx, flow, managementreg, ucast0, \
+    ucast1, addrtable0, addrtable1, addrfiltermode, reserved = range(11)
+
+
+class mdioData:
+    def __init__(self):
+        self.hostclk_count = Signal(intbv(0)[5:])  # Used to drive mdc
+        self.wrdata = Signal(intbv(0)[32:])
+        self.rddata = Signal(intbv(0)[16:])
+        self.rdindex = Signal(intbv(17, min=0, max=18))  # in index during rdop
+        self.wrindex = Signal(intbv(62, min=0, max=63))
+        self.wrdone = Signal(bool(0))
+        self.done = Signal(bool(0))
+
+"""
```

Comment:
Use the google style string docs as discussed in the [best practices]()  (see the Documentation section).  Update the docstring to match the changes.